tensorflow_probability/python/bijectors/conditional_bijector_test.py

```diff
@@ -48,6 +48,32 @@ def _forward_log_det_jacobian(self, _, arg1, arg2):
     raise ValueError("forward_log_det_jacobian", arg1, arg2)
 
 
+class _TestPassthroughBijector(_TestBijector):
+  def __init__(self, *args, **kwargs):
+      super(_TestPassthroughBijector, self).__init__(*args, **kwargs)
+      self._called = {
+          name: False for name in
+          ('forward', 'inverse',
```

Comment:
Would you mind using " instead of ', for consistency with the rest of the file (our internal lints don't like us being inconsistent).